SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -4,14 +4,14 @@
 
 Name:           cloud-hypervisor-cvm
 Summary:        Cloud Hypervisor CVM is an open source Virtual Machine Monitor (VMM) that enables running SEV SNP enabled VMs on top of MSHV using the IGVM file format as payload.
-Version:        38.0.72
+Version:        38.0.72.2
 Release:        1%{?dist}
 License:        ASL 2.0 OR BSD-3-clause
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Applications/System
 URL:            https://github.com/microsoft/cloud-hypervisor
-Source0:        https://github.com/microsoft/cloud-hypervisor/archive/refs/tags/msft/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source0:        https://lsgvirtstorage.blob.core.windows.net/dom0-mariner-sources-public/cloud-hypervisor-%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

Comment:
We should use the github tagged release version - https://github.com/microsoft/cloud-hypervisor/releases/tag/msft%2Fv38.0.72.2